pkg/kubectl/kubectl_test.go

```diff
@@ -142,17 +142,23 @@ func TestParseLiteralSource(t *testing.T) {
 		{
 			name:  "err 1",
 			input: "key==value",
-			err:   true,
+			key:   "key",
+			value: "=value",
+			err:   false,
 		},
 		{
 			name:  "err 2",
 			input: "key=value=",
-			err:   true,
+			key:   "key",
```

Comment:
The root bug was the user needed the secret to store a base64 encoded value because the application consumed base64.  So for example, "haha" encodes as "aGFoYQ==", the user should be able to say key=aGFoYQ== because "aGFoYQ==" is what his application is consuming.  Internally, the secret will store the double encoded form, YUdGb1lRPT0=